homework/03LessonThree.js

```diff
@@ -89,17 +95,19 @@ var problemSevenAnswer;
 
 // Problem 8:
 // Create an empty array
-var emptyArray;
+var emptyArray = [ ];
 
 // Problem 9:
 // Create an array with 5 elements in it
-var lengthFiveArray;
+var lengthFiveArray = [ "Yoda", "Luke", "Leia", "Darth Vader", "Obi-Wan"];
 
 // DO NOT MODIFY
 var nameArray = ['bob', 'fred', 'susan'];
 
 // Problem 10:
 // Replace the value 'fred' in nameArray with 'george'
+nameArray[2] = george;
```

Comment:
fixed this error in latest commit <---- that sounds prettttay programmer-y. So proud of my newfound jargon skillz.